vector/src/main/java/im/vector/app/features/onboarding/OnboardingAction.kt

```diff
@@ -31,6 +31,15 @@ sealed class OnboardingAction : VectorViewModelAction {
 
     data class UpdateServerType(val serverType: ServerType) : OnboardingAction()
     data class UpdateHomeServer(val homeServerUrl: String) : OnboardingAction()
+    data class UpdateUseCase(val useCase: UseCase) : OnboardingAction() {
+        enum class UseCase {
```

Comment:
Nit: Other enums (`SignMode`, etc.) are declared in their own file, maybe do the same, this file is big enough.

Reply:
extracted 5d0c55b6179cc8335ee3ab2cc16322ff7a687c12